### DISCUSSION TITLE
Step 5 in documenting the trainings is obsolete

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#67] [Shipped - Live on production]
Step 5 (Confirm training) is unnecessary for digitally documented briefings.

https://user-images.githubusercontent.com/80519295/231502759-dd4a41b7-4e47-46fd-89a2-bacfe2f79df6.mov

In the case of trainings documented in paper form, quality control may be useful or required. The rights concept does currently not provide for a separate "quality check" role. There is only the right "Set device-model trainings invalid".

Currently, anyone who can document trainings can also confirm them. This means that step 5 is actually obsolete.

**Comment by YvesR** [comment]
The next release will do the approve step only if it is in paper-form, digital trainings will skip this.
Also, we add the access management cando so it can be defined who will review and approve the last step.
